README.md

```diff
@@ -62,76 +210,102 @@ First ensure you are in a new and empty directory.
+
+```shell
+truffle migrate
+```
+4. Update Express JS component. 
+
+The express server uses the [web3.js](https://web3js.readthedocs.io/) library to interact with the blockchain - writing code that reads and writes data from the blockchain with smart contracts.
+
+Choose which network you would like to connect the server to RSK Network 
+and update line 11 of the file `app/src/server/index.js`
+
+**Testnet**
+```js    
+const provider = new Web3.providers.HttpProvider("https://public-node.testnet.rsk.co");    
+```
 
-    const HDWalletProvider = require('@truffle/hdwallet-provider');
+**Mainnet**
+```js        
+const provider = new Web3.providers.HttpProvider("https://public-node.rsk.co");
+```
 
-    //Put your mnemonic here, be careful not to deploy your mnemonic into production!
-    const mnemonic = 'A_MNEMONIC';
-    ```
-    Please be aware that we are using `HDWalletProvider` with RSK Networks derivations path:
-    - RSK Mainnet dpath: `m/44’/137’/0’/0`
-    - RSK Testnet dpath: `m/44’/37310’/0’/0`
+5. In a terminal, go to the `app` directory and run the app.
 
-    For more information check [RSKIP57](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP57.md).
+For example, this command will run the production server:
 
-2. Check the gas price of the network, and update `truffle-config.js` if necessary.
+```shell
+cd app
+npm start
+```
 
-3. Check Express JS component. If you want to connect the server to RSK Network, update line 11 of `app/src/server/index.js`
-    **Mainnet**
-    ```js        
-    const provider = new Web3.providers.HttpProvider("https://public-node.rsk.co");
-    ```
+Then go to your browser at [http://localhost:8080/](http://localhost:8080/)
 
-    **Testnet**
-    ```js    
-    const provider = new Web3.providers.HttpProvider("https://public-node.testnet.rsk.co");    
-    ```
+> Note that when you are connected to an RSK network, you do not need to leave open the Truffle console, because the app is connected via a public node, directly to the network.
 
-4. Run the development console for any RSK network.
+## Next steps
 
-    ```shell
-    # Console for Mainnet
-    truffle console --network mainnet
+- **Go to tutorial**
 
-    # Console forn Testnet
-    truffle console --network testnet
-    ```
+Go to the tutorial [using rsk-react-express-box](https://developers.rsk.co/tutorials/truffle-boxes/rsk-react-express-box/) to learn more about this project. We covered all the steps with more details, explanations, and images.
 
-5. Compile and migrate the smart contracts. Note that inside the development console, we don't preface commands with truffle.
+- **Find more documentation**
 
-    ```shell
-    compile
-    migrate
-    ```
+Check out the [RSK developers portal](https://developers.rsk.co/).
 
-**Then continue from step 5 of [installation steps](#installation)**
+- **Do you have questions?**
 
+Ask in [RSK chat](https://gitter.im/rsksmart/getting-started).
```

Comment:
```suggestion
Ask in the [RSK Gitter Chat](https://gitter.im/rsksmart/getting-started).
```

Reply:
No, I wouldn't like to specify the name of the chat.

Reply:
```suggestion
Ask in the [RSK chat](https://gitter.im/rsksmart/getting-started).
```